### ISSUE TITLE
Execute code server-side

### ISSUE THREAD

**Issue by ajuancer** [wikimedia/WikiContrib#103]
Hi!
In issue #101 Can't use FS functions, the given solution is to execute that fs-dependent function in server-side instead of client-side. How can I do that?

**Comment by rammanoj** [comment]
Can I know the reason why you want to execute FS functions ? Are you trying to implement any functionalities ? If so, can I know them, so that I can help you more...

As far as I know, the server side programming language is python. You can find it in backend/ directory.. You can use ``fs`` equivalent function there.

**Comment by ajuancer** [comment]
Thanks! Yes, I see the backend folder but I thought that maybe I missed a subfolder that contains JS. There's no problem for me to implement the function in python, but I'm not sure how to use the function (import it, call it with the needed attribute and use its return) inside the JS file. The function returns a string (that would be used in `activity.js`. Another question is in which file should I implemented it? I suppose in one file of the`result` folder.

**Comment by rammanoj** [comment]
Can I know what do you want to implement?

**Comment by ajuancer** [comment]
Of course! I'm trying to show an icon in the user activity on a specific date, similar to issue #71 . This icon is an SVG file. The objective is to return the corresponding SVG depending on the status. The code (already in python) is:
```python
def selectStatusIcon(status):
    path = "";
    status = status.lower();
    if (status == "merged"):
      path = 'mergedStatusIcon.svg';
    elif (status == "open"):
      path = 'openStatusIcon.svg'
    else:
      path = 'abandonedStatusIcon.svg'
    selectedFile = open(path, 'r')
    if (selectedFile.mode == 'r'):
        content = selectedFile.read();
    # print(content);
    return content;
```
I have tested and it works fine, but I'm not sure how to use it in `activity.js`. 
Thanks!

**Comment by rammanoj** [comment]
@ajuancer sorry for delay.

Why do you want to read the image ? For displaying the image, you can just write the following code:
```html
<img src={URL} />
```

I recommend you to **write a function that return the path of the url of the image** so that the returned url can be fitted in the above code. If you are just trying to show the icon, there is no need to read the file, just return the path of the file to the html ``img`` tag.

**Comment by ajuancer** [comment]
Hi! Answering your question, I'm interested in returning an SVG, not a normal image. Issue #71, that works with images, is already resolved (since it's more related CGI information I wrote it in the state of the task), but I decided to make some icons instead of showing the status's name. As SVG is better than a normal img, I designed them in this format, but, anyway, as I haven't much more time, I have just exported them as normal images. Thanks for the answer!
